pkg/tools/fit.go

```diff
@@ -57,15 +58,15 @@ func (fit *FitEntry) FancyPrint() {
 	log.Println("Fit Table PrintOut")
 	if fit.Address == type0MagicWord {
 		log.Println("FitEntry 0")
-		log.Printf("Fit Size: %v\n Entries", fit.Size())
-		log.Printf("Version: %v\n", fit.Version)
-		log.Printf("Checksum indicator: %b\n", fit.CVType)
+		log.Printf("Fit Size: 0x%x\n Entries", fit.Size())
+		log.Printf("Version: 0x%x\n", fit.Version)
+		log.Printf("Checksum indicator: 0x%x\n", fit.CVType)
 	} else {
-		log.Printf("Component Address: %v\n", fit.Address)
-		log.Printf("Component size: %v\n", fit.Size())
-		log.Printf("Version: %v\n", fit.Version)
-		log.Printf("C_V & Type: %b\n", fit.CVType)
-		log.Printf("Checksum: %v\n", fit.CheckSum)
+		log.Printf("Component Address: 0x%x\n", fit.Address)
```

Comment:
missing Fit type, without that it's just a list of magic numbers

Reply:
We'll fix that when it's merged